user-guide/report-server-manager-overview.md

```diff
@@ -0,0 +1,71 @@
+position: 800
+---
+
+# Report Server Manager
+
+The Report Server Manager is the ASP.NET Application that lets the Report Server users login and use its functionality through a user-friendly interface. It provides a dedicated view for each functionality of the server. These views are explained in detail in the corresponding articles:
+
+* [Reports Management]({%slug reports-management%})
+
+* [Categories Management]({%slug categories-management%})
+
+* [Data Connections Management]({%slug data-connections-management%})
+
+* [Scheduled Tasks Management]({%slug tasks-management%})
+
+* [Data Alerts Management]({%slug alerts-management%})
+
+* [Webhooks Management]({%slug webhooks-management%})
+
+* [Users]({%slug users%})
+
+* [User Roles]({%slug user-roles%}) 
+
+The purpose of this article is to describe the common functionality of the Report Server views that would let you access and organize your Report Server assest easier. This includes [Sorting](#Sort), [Page Size](#page-size) and [Searching](#search).
+
+
+## Sort
+
+The Report Server allows you to sort the items in each view by the grid columns in _Ascending_ and _Descending_ order. The views support multi-column sorting. The order is displayed with an arrow on the right of the column name. The upward arrow indicates _Ascending_ order, the downward arrow indicates _Descending_ order. If there is no user-defined sorting, the default sorting gets applied. The next table describes the default sorting for the views:
+
+>caption Default Sorting
+
+| View | Primary field | Secondary field | Tertiary field |
+|---|---|---|---|
+|__Reports__| Favorite | Date Modified | Name |
+|__Categories__| Name | | |
+|__Data__| Name | | |
+|__Scheduling__| Enabled | Next Occurance | Name |
+|__Data Alerts__| Enabled | Last Run | Name |
+|__Users__| Username | | |
+|__Roles__| Name | | |
+|__Webhooks__| Webhook URL | | |
+
+The items' order in the multi-column sorting depends on the sequence in which you have applied sorting to the columns. The sorting applied earlier is prioritized. For example, if you sort the Reports view first by `Category`, and then by `Name/Description`, the grid will sort the items by `Category` and then would apply the sorting within each category by report `Name/Description` as shown in the next image. 
+
+>caption __Sorting__ and __Page Size__ in Report Server Grid Views
+
+![Sorting and Page Size](../../images/report-server-images/sorting-page-size.png)
+
+If you sort first by `Name/Description`, the grid will sort the items by this column, and when you apply a secondary sorting by `Category` it would be applied to the already sorted list. This may have effect only when there are reports with the same name in different categories.
+
+The Sorting is kept in the [browser's Window Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage). This means that the sorting is applied for each client individually. The local storage property name is `sortArray`.
```

Comment:
I guess you refer to clients meaning the web browser but it is a bit confusing to me with the clients of the RS product. Lets talk to the users of the product instead? How about: The Sorting is kept in the [browser's Window Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) in a property called `sortArray`. This way the preferred sort rules get preserved individually for each user within their web browser.